target/linux/pistachio/base-files/lib/netifd/proto/lowpan.sh

```diff
@@ -34,6 +34,7 @@ proto_lowpan_setup() {
 
         proto_add_ipv6_address "$ip6addr" "128"
         proto_send_update "$cfg"
+        sync
```

Comment:
I don't think its changing the network config file. rather you need to add sync in etc/uci_defaults/01_default_config where the file gets copied.